src/wavm/openmp.cpp

```diff
@@ -512,7 +508,7 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env,
         setUpOpenMPContext(0, nextLevel);
 
         // Execute the task
-        logger->debug("Executing master OMP thread");
+        logger->debug("OpenMP 0: executing OMP thread 0 (master)");
```

Comment:
Just thinking out loud here, but you could have a `thread local` logger, and use something like:
```cpp
auto logger = faabric::util::getLogger(fmt::format("OMP-{}", threadIdx));
```
This is not a request for changes, rather a suggestion if you find it useful.

Reply:
Hmm, I wonder what's better in terms of contention, a single logger used by all threads or a logger per thread? I would go with whatever is more performant, my gut being a single logger used by all threads, but I may be wrong.

Reply:
Well, you are probably right. But one could argue that when measuring performance logging will be turned off.